src/site/config.toml

```diff
@@ -6,38 +6,38 @@ title = "ServerlessDays Melbourne"
     logo = "/images/serverless_logo_melb.png"
     description = "A community based conference focusing on Serverless based platforms and technologies. Melbourne, August 29, 2019"
```

Comment:
We should also change it in here the 2019 description to "August 20th, 2020" ?

Reply:
Fixed :)